### PR TITLE
app/vmui: added table where Labels with the highest number of unique values show

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Added table with the highest number of unique values. This table hasn't any actions because we can't construct the correct request to show information by metrics. But we can find information about these labels through the new flow of the exploring metrics.

Related issue: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4213

Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

File: app/vmui/packages/vmui/src/pages/CardinalityPanel/Table/TableCells/TableCells.tsx [hunk 1]
```diff
@@ -25,9 +25,27 @@ const TableCells: FC<CardinalityTableCells> = ({
 
   const diffPercent = progress - progressPrev;
   const relationPrevDay = hasProgresses ? "" : `${diffPercent.toFixed(2)}%`;
+  const hasActions = !!onActionClick;
 
   const handleActionClick = () => {
-    onActionClick(row.name);
+    if (hasActions) {
+      onActionClick(row.name);
```
Comment (Loori-R) [inline app/vmui/packages/vmui/src/pages/CardinalityPanel/Table/TableCells/TableCells.tsx:32]: I think `hasActions` is excessive

```
 const handleActionClick = () => {
      onActionClick && onActionClick(row.name);
  };
```

File: app/vmui/packages/vmui/src/pages/CardinalityPanel/Table/TableCells/TableCells.tsx [hunk 2]
```diff
@@ -90,15 +108,7 @@ const TableCells: FC<CardinalityTableCells> = ({
...
-        <Tooltip title={`Filter by ${row.name}`}>
-          <Button
-            variant="text"
-            size="small"
-            onClick={handleActionClick}
-          >
-            <PlayCircleOutlineIcon/>
-          </Button>
-        </Tooltip>
+        {actionButton()}
```
Comment (Loori-R) [inline app/vmui/packages/vmui/src/pages/CardinalityPanel/Table/TableCells/TableCells.tsx:111]:
```
{onActionClick && (
  <Tooltip title="{`Filter" by ${row.name}`}>
    <Button 
      variant="text" 
      size="small" 
      onClick="{handleActionClick}"
      >
      <PlayCircleOutlineIcon />
    </Button>
  </Tooltip>
)}
```

**Review by Loori-R — COMMENTED** [review]
Please see comments

File: app/vmui/packages/vmui/src/pages/CardinalityPanel/Table/TableCells/TableCells.tsx (hunk 1 above)
Comment (dmitryk-dk) [reply app/vmui/packages/vmui/src/pages/CardinalityPanel/Table/TableCells/TableCells.tsx]: updated

**Review by Loori-R — APPROVED** [review]
LGTM!

**Comment by hagen1778** [comment]
It is unclear what should happen when clicking on the label name in table:
image

**Comment by dmitryk-dk** [comment]
> It is unclear what should happen when clicking on the label name in table: image

Removed link classes if actions is null

**Comment by dmitryk-dk** [comment]
> > Removed link classes if actions is null
> 
> I was thinking maybe on click we can check all the values of this label? Or should it be done in other way?

Rollback actions and shows the label values for selected label with the highest number of series
Screenshot 2023-05-10 at 15 14 53

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -47,6 +47,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
 * FEATURE: [vmbackup](https://docs.victoriametrics.com/vmbackup.html): store backup creation and completion time in `backup_complete.ignore` file of backup contents. This allows determining the exact timestamp when the backup was created and completed.
 * FEATURE: [vmbackupmanager](https://docs.victoriametrics.com/vmbackupmanager.html): add `created_at` field to the output of `/api/v1/backups` API and `vmbackupmanager backup list` command. See this [doc](https://docs.victoriametrics.com/vmbackupmanager.html#api-methods) for data format details.
 * FEATURE: [vmctl](https://docs.victoriametrics.com/vmctl.html): add support for [different time formats](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#timestamp-formats) for `--vm-native-filter-time-start` and `--vm-native-filter-time-end` command-line flags. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4091).
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): rollback table with the highest number of unique values in the main page. This table helps to find labels with huge unique values. See [issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4213).
```
Comment (hagen1778) [inline docs/CHANGELOG.md:50]:
```suggestion
* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add table with the highest number of unique label values. This table helps to determine labels with a big number of unique values. See [issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4213).
```

**Review by hagen1778 — APPROVED** [review]
small fix

**Pull request merged**